### ISSUE TITLE
AsyncEth object has no attribute 'contract'

### ISSUE THREAD

**Issue by DevTenset** [ethereum/web3.py#2490]
* Version: x.x.x
* Python: 3.8
* OS: osx
* 
### What was wrong?

AttributeError: 'AsyncEth' object has no attribute 'contract'

```
contract = w3.eth.contract(address=Web3.toChecksumAddress(contract["address"]), abi=abi.read())
contract.events.Transfer.createFilter(...)
```

### How can it be fixed?

Contract create event class which call to Filter class, and this call finally to `self.eth_module.get_logs(self.filter_id)` so just add support for Contract class, because for now I have to do manually with get_logs, and I have raw data

---

**Comment by pacrob** [comment]
It's done! See PR #2484. It's been merged in to master so you can use it from there. A release containing it will come in the next week or two. Filters are not yet asynced, that will be in the next month or two.

**Comment by 0xjkrdev** [comment]
Hello,

I have the same error : AttributeError: 'AsyncEth' object has no attribute 'contract'
I am using web3 version 5.29.2
Has the fix been released ?

Thanks for your help,

**Comment by kclowes** [comment]
@0xjkrdev it was released in v6.0.0-beta.3. You'll have to try it out there for now.

**Comment by 0xjkrdev** [comment]
@kclowes Awesome, it's working.
Thanks a lot for your quick answer.
